docs/console/game/balancing/game_config/index.md

```diff
@@ -0,0 +1,78 @@
+# Game Config
+
+This page shows the existing Game Configurations. In this page you can create a new Game config or
+edit/activate/duplicate/view/remove an existing one. Game configs that are already been active cannot be edited or
+removed.
+
+## Contents
```

Comment:
I would prefer to have 'a' inserted in the titles.. eg. Create a new Game Config etc.... (also for SDK Config)